pkg/gnmi/util.go

```diff
@@ -346,12 +346,12 @@ func (s *Server) sendResponse(response *pb.SubscribeResponse, stream pb.GNMI_Sub
 // getUpdateForPath finds a leaf node in the tree based on a given path, build the update message and return it back to the collector
 func (s *Server) getUpdateForPath(fullPath *pb.Path) (*pb.Update, error) {
 
-	node, stat := ygotutils.GetNode(s.model.schemaTreeRoot, s.config, fullPath)
-	if isNil(node) || stat.GetCode() != int32(cpb.Code_OK) {
-		return nil, status.Errorf(codes.NotFound, "path %v not found", fullPath)
+	node, err := ytypes.GetNode(s.model.schemaTreeRoot, s.config, fullPath, nil)
```

Comment:
should the test for `nil` and `Code != OK` be kept here

Reply:
Good catch. Yeah I still need to check all of the errors handled. In the new API, it doesn't return the stat but returns an error. We just need to check if node is nil or err != nil then return nil,err